lib/member_page.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class MemberPage < Scraped::HTML
+  field :phone do
+    box.xpath('.//span[@class="item"][contains(.,"Phone")]//following-sibling::text()')
+       .text
+       .tidy
+  end
+
+  field :email do
+    box.xpath('.//span[@class="item"][contains(.,"Email")]//following-sibling::text()')
+       .text
+       .tidy
+  end
+
+  field :birth_date do
+    # At least one member has an invalid birth date of 00-00-00.
+    date_str = box.xpath('.//span[@class="item"][contains(.,"Date of Birth")]//following-sibling::text()').text.tidy
+    if Date.valid_date?(*date_str.split('-').map(&:to_i))
```

Comment:
I'm not 100% convinced by this approach, as it's going to raise an error (rather than just return `false`) if `date_str` is in the wrong format, including if its an empty string. But until we see an example of that, I think this is enough of an improvement to go live.